src/libraries/System.Private.CoreLib/src/System/Resources/ManifestBasedResourceGroveler.cs

```diff
@@ -22,15 +10,18 @@
 
 namespace System.Resources
 {
-    //
-    // Note: this type is integral to the construction of exception objects,
-    // and sometimes this has to be done in low memory situtations (OOM) or
-    // to create TypeInitializationExceptions due to failure of a static class
-    // constructor. This type needs to be extremely careful and assume that
-    // any type it references may have previously failed to construct, so statics
-    // belonging to that type may not be initialized. FrameworkEventSource.Log
-    // is one such example.
-    //
+    /// <summary>
+    /// Searches for resources in Assembly manifest, used for assembly-based resource lookup.
+    /// </summary>
+    /// <remarks>
+    /// Note: this type is integral to the construction of exception objects,
+    /// and sometimes this has to be done in low memory situtations (OOM) or
+    /// to create TypeInitializationExceptions due to failure of a static class
+    /// constructor. This type needs to be extremely careful and assume that
+    /// any type it references may have previously failed to construct, so statics
+    /// belonging to that type may not be initialized. FrameworkEventSource.Log
+    /// is one such example.
+    /// </remarks>
```

Comment:
These seem to be maintainer comment, so double slash is more appropriate. Those also don't need the usage of crefs.

```suggestion
    // Note: this type is integral to the construction of exception objects,
    // and sometimes this has to be done in low memory situtations (OOM) or
    // to create TypeInitializationExceptions due to failure of a static class
    // constructor. This type needs to be extremely careful and assume that
    // any type it references may have previously failed to construct, so statics
    // belonging to that type may not be initialized. FrameworkEventSource.Log
    // is one such example.
```